docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,1903 @@
+{
```

Comment:
<p>It looks you may have <code>ipywidgets</code> installed, which apparently allows the progress bar to work. Any chance you can run the notebook without that so there's <em>only</em> the number showing up in the output?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
`ipywidgets` requires JavaScript to properly show up, so I'd rather leave it out if we can.

Reply:
Yeah, I got around the bug by installing ipyidgets. But I think you're right. I should just update ibis to main and then rerun this without the progress bar. Will do!

Reply:
<p>Done in v4</p>